modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -47,7 +47,7 @@ To expose the registry using custom routes:
 +
 ----
 $ oc create secret tls public-route-tls \
-    -n image-registry \
+    -n openshift-image-registry \
     --cert=</path/to/tls.crt> \
```

Comment:
How did this get into your PR?

Reply:
I'm also unsure how this got here, but this does reflect what we have in master. Added in this PR https://github.com/openshift/openshift-docs/pull/22118

Reply:
OK, that explains why i suddenly had "namespace fix" in the list of recent commits I saw for this branch.

Reply:
It's a bug. I've noticed that it happens quite a lot lately. There is a way to fix it when a new branch has other people's commits (force reset head, CP your commit).